### ISSUE TITLE
Expose a super-simple POI editor instead of asking questions in email

### ISSUE THREAD

**Issue by iandees** [osmlab/checkin-checker#21]
Right now we link to the user's editor on OSM (e.g. https://www.openstreetmap.org/edit?way=20967912) when the checkin venue matches something in OSM. In most cases, this will lead to the iD editor. Since people will very likely be on their phone when they checkin and receive the email, pointing them to iD is a terrible experience.

Instead, let's set up something like [Poi Pond](https://github.com/mapzen/poipond) or a similar mobile-friendly interface to collect more data.

**Comment by wboykinm** [comment]
A non-native version of [this](https://itunes.apple.com/us/app/pushpin-osm/id565850639?mt=8), mayhap? Is there a working version of POI Pond out there anywhere to look at?

**Comment by iandees** [comment]
Yep, something like Pushpin. I don't think there's a POI Pond set up anywhere. I'm working on getting an instance of it set up right now.

Another thing I'd like to consider is doing something like Google Maps' yes/no/don't know interface:

![image](https://cloud.githubusercontent.com/assets/261584/19251936/ea38688e-8f0f-11e6-9134-9861fa3f3d47.png)

…with a possible extension for simple text input (address/phone number).
